packages/react/src/components/SuiteHeader/hooks/useSuiteHeaderData.js

```diff
@@ -1,3 +1,5 @@
+// This custom hook is deprecated, use useUiResources instead
```

Comment:
If you move this to a docblock statement on the function, then IntelliSense will be able to pick it up.

```
/**
 * @deprecated use useUiResources instead
 */
const useSuiteHeaderData = ({
```
![Screen Shot 2021-05-18 at 9 47 24 AM](https://user-images.githubusercontent.com/609466/118662808-4be68580-b7be-11eb-9883-297fa5822d37.png)

Reply:
Thanks, I didn't know about this `@deprecated` annotation. I'll make the change